applications/fishing-map/src/features/sidebar/Sidebar.tsx

```diff
@@ -28,7 +30,7 @@ function SidebarHeader({ onMenuClick }: SidebarProps) {
       <div className={styles.sidebarHeader}>
         <IconButton icon="menu" onClick={onMenuClick} />
         <Logo className={styles.logo} />
-        <IconButton icon="share" tooltip="Click to share this view" tooltipPlacement="bottom" />
+        <IconButton icon="share" tooltip={t('tooltips.share')} tooltipPlacement="bottom" />
```

Comment:
I don't see the string translated in english nor in spanish checked in, am I missing something?

Reply:
Didn't see the other PR... Curious how it works

Reply:
Because the translations files are in this PR https://github.com/GlobalFishingWatch/frontend/pull/158